src/modules/modal.js

```diff
@@ -57,7 +58,7 @@ const mainTitle = (tag, id) => {
   return header;
 };
 
-const createModal = () => {
+const createModal = (artistId) => {
   const commentWrapper = createDiv(['comment-wrapper']);
   const artistImage = createDiv(['artist-image']);
 
```

Comment:
Great job adding the functionality to create the modal dynamically @wayungi :+1: